core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java

```diff
@@ -820,12 +820,12 @@ public void setAckMode(int ackMode) {
 		this.ackMode = EnumUtils.parseFromField(AcknowledgeMode.class, "ackMode", ackMode, a -> a.getAcknowledgeMode());
 	}
 
-	public AcknowledgeMode getAckModeEnum() {
+	public AcknowledgeMode getAcknowledgeModeEnum() {
 		return ackMode;
 	}
 
 
-	@IbisDoc({"3", "Acknowledge mode, can be one of ('auto' or 'auto_acknowledge'), ('dups' or 'dups_ok_acknowledge') or ('client' or 'client_acknowledge')", "auto_acknowledge",})
+	@IbisDoc({"3", "Acknowledge mode", "auto"})
```

Comment:
`If not transacted, the way the application informs the JMS provider that it has successfully received a message.`

Reply:
Gedaan.